zyte_spider_templates/settings.py

```diff
@@ -0,0 +1,42 @@
+from itemadapter import ItemAdapter
```

Comment:
What's the use case for this file?

Reply:
Good catch! This should not have been included as it's used locally for testing.